lib/domain/board.rb

```diff
@@ -5,3 +5,14 @@ def initialize(size:)
     size.times { push(nil) }
   end
 end
+
+WINNING_COMBINATIONS = [
```

Comment:
It's interesting to me that `WINNING_COMBINATIONS` in placed outside of the class. Is there a particular reason why?

If not, as it's related to the `Board` class I would suggest making it a class constant. I'm thinking to move it in-between lines 3 and 4 so before the `initialize` method. If you do this, you'll then have to call it like so: `Board::WINNING_COMBINATIONS`.